HRA tests failing in actions

I'm seeing some HRA tests fail in an auto PR for merging `main` into `release/3.10.2. These tests must not have been failing when merging into `main`.  They are failing for Windows and Mac Python 3.8, 3.9. Python 3.7 tests pass okay. https://github.com/natcap/invest/pull/832

This line in HRAs `_zonal_stats_to_csv` function appears to be the culprit:
`    stats_df = pandas.DataFrame(index=overlap_df.index, columns=columns)`

```
2022-01-25T19:17:25.4015294Z =========================== short test summary info ===========================
2022-01-25T19:17:25.4015663Z FAILED tests/test_hra.py::HraRegressionTests::test_hra_no_subregion_multiplicative_exponential
2022-01-25T19:17:25.4016097Z FAILED tests/test_hra.py::HraRegressionTests::test_hra_regression_euclidean_linear
2022-01-25T19:17:25.4016474Z ====== 2 failed, 585 passed, 8 skipped, 15 warnings in 457.84s (0:07:37) ======
```

```
2022-01-25T19:12:49.9538188Z tests/test_hra.py::HraRegressionTests::test_aoi_no_projection PASSED
2022-01-25T19:12:56.0746787Z tests/test_hra.py::HraRegressionTests::test_hra_no_subregion_multiplicative_exponential 01/25/2022 19:12:56  taskgraph.Task     ERROR    Something went wrong when adding task zonal_stats_to_csv (55), terminating taskgraph.
2022-01-25T19:12:56.0747696Z Traceback (most recent call last):
2022-01-25T19:12:56.0748148Z   File "C:\Miniconda\envs\windows-latest-test-env\lib\site-packages\taskgraph\Task.py", line 674, in add_task
2022-01-25T19:12:56.0752229Z     new_task._call()
2022-01-25T19:12:56.0752790Z   File "C:\Miniconda\envs\windows-latest-test-env\lib\site-packages\taskgraph\Task.py", line 1093, in _call
2022-01-25T19:12:56.0753434Z     payload = self._func(*self._args, **self._kwargs)
2022-01-25T19:12:56.0754009Z   File "C:\Miniconda\envs\windows-latest-test-env\lib\site-packages\natcap\invest\hra.py", line 1186, in _zonal_stats_to_csv
2022-01-25T19:12:56.0754603Z     stats_df = pandas.DataFrame(index=overlap_df.index, columns=columns)
2022-01-25T19:12:56.0755163Z   File "C:\Miniconda\envs\windows-latest-test-env\lib\site-packages\pandas\core\frame.py", line 637, in __init__
2022-01-25T19:12:56.0755760Z     mgr = dict_to_mgr(data, index, columns, dtype=dtype, copy=copy, typ=manager)
2022-01-25T19:12:56.0756750Z   File "C:\Miniconda\envs\windows-latest-test-env\lib\site-packages\pandas\core\internals\construction.py", line 502, in dict_to_mgr
2022-01-25T19:12:56.0758177Z     return arrays_to_mgr(arrays, columns, index, dtype=dtype, typ=typ, consolidate=copy)
2022-01-25T19:12:56.0758923Z   File "C:\Miniconda\envs\windows-latest-test-env\lib\site-packages\pandas\core\internals\construction.py", line 150, in arrays_to_mgr
2022-01-25T19:12:56.0759763Z     raise ValueError("len(arrays) must match len(columns)")
2022-01-25T19:12:56.0760244Z ValueError: len(arrays) must match len(columns)
```

--- a/HISTORY.rst
+++ b/HISTORY.rst
@@ -48,6 +48,10 @@ Unreleased Changes
       one stream pixel, it will now snap to the stream pixel with a higher
       flow accumulation value. Before, it would snap to the stream pixel
       encountered first in the raster (though this was not guaranteed).
+* HRA
+    * Fixed a bug with how a pandas dataframe was instantiated. This bug did
+      not effect outputs though some might notice less trailing zeros in the
+      ``SUMMARY_STATISTICS.csv`` output.
 * NDR
     * Changed some model inputs and outputs to clarify that subsurface
       phosphorus is not modeled.

--- a/src/natcap/invest/hra.py
+++ b/src/natcap/invest/hra.py
@@ -1183,7 +1183,7 @@ def _zonal_stats_to_csv(
         ['E_']*len_crit_cols + ['C_']*len_crit_cols + ['R_']*len_risk_cols,
         crit_stats_cols*2 + risk_stats_cols)
 
-    stats_df = pandas.DataFrame(index=overlap_df.index, columns=columns)
+    stats_df = pandas.DataFrame(index=overlap_df.index, columns=list(columns))
 
     # Add a ``SUBREGION`` column to the dataframe and update it with the
     # corresponding stats in each subregion

